app/graphql/types/enum/locked_reason.rb

```diff
@@ -0,0 +1,4 @@
+class Types::Enum::LockedReason < Types::Enum::Base
+  value 'SPAM', value: 'spam'
```

Comment:
Closure for community events (or ancient Kitsu group posts?...those half year old posts are still 95% of my notifications)

```suggestion
  value 'CLOSED', value: 'closed'
  value 'SPAM', value: 'spam'
```